CMakeLists.txt

```diff
@@ -388,7 +388,7 @@ else()
       set(CHPL_INSTALL_RPATH "@executable_path/../../lib/compiler/${CHPL_HOST_BIN_SUBDIR}")
     endif()
 
-    set_target_properties(chpl chpldoc
+    set_target_properties(chpl chpldoc ChplFrontendShared
```

Comment:
does this need to be aligned with `prefix` install mode on line 369?

Reply:
Good point, I think it does.